plugins/outputs/dynatrace/README.md

```diff
@@ -1,53 +1,120 @@
+### Running alongside Dynatrace OneAgent
+
+If you run the Telegraf agent on a host or VM that is monitored by the Dynatrace OneAgent then you only need to enable the plugin, but need no further configuration. The Dynatrace telegraf output plugin will send all metrics to the OneAgent which will use its secure and load balanced connection to send the metrics to your Dynatrace SaaS or Managed environment.
+
+note: The name and identifier of the Telegraf host will be added as a dimension to every metric. If this is undesireable, then the output plugin may be used in standalone mode using the directions below.
+
+```toml
+[[outputs.dynatrace]]
+  ## No options are required. By default, metrics will be exported via the OneAgent on the local host.
+```
 
 ## Running standalone
 
 If you run the Telegraf agent on a host or VM without a OneAgent you will need to configure the environment API endpoint to send the metrics to and an API token for security.
 
-The endpoint for the Dynatrace Metrics API is 
+You will also need to configure an API token for secure access. Find out how to create a token [here](https://www.dynatrace.com/support/help/dynatrace-api/environment-api/tokens/) or simply navigate to **Settings > Integration > Dynatrace API** in your Dynatrace environment and create a token with Dynatrace API and create a new token with 
+'Ingest metrics' scope enabled. 
 
-* Managed https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest
-* SaaS https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest
+The endpoint for the Dynatrace Metrics API v2 is 
 
-You can learn more about how to use the Dynatrace API [here](https://www.dynatrace.com/support/help/dynatrace-api/)
+* Managed `https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest`
+* SaaS `https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest`
 
-You will also need to configure an API token for secure access. Find out how to create a token [here](https://www.dynatrace.com/support/help/dynatrace-api/environment-api/tokens/) or simply navigate to **Settings > Integration > Dynatrace API** in your Dynatrace environment and create a token with Dynatrace API and create a new token with 
-'Ingest metrics data points' access scope enabled. 
+```toml
+[[outputs.dynatrace]]
+  ## If no OneAgent is running on the host, url and api_token should be set
+
+  ## Dynatrace Metrics Ingest v2 endpoint to receive metrics
+  url = "https://YOUR_DOMAIN/api/v2/metrics/ingest"
+
+  ## API token is required if a URL is specified and should be restricted to the 'Ingest metrics' scope
+  api_token = "your API token here"
+```
+
+You can learn more about how to use the Dynatrace API [here](https://www.dynatrace.com/support/help/dynatrace-api/)
 
 ## Configuration
 
+### `url`
+
+*required*: `false`
```

Comment:
```suggestion
*required*: `false`  
```
to force a linebreak

Reply:
I would rather not depend on trailing whitespace

```suggestion
*required*: `false`

```